docs/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Added
 - Implement major_incident_state in Flaw API (OSIDB-266)
+- Add CVSS model for Flaws and Affects (OSIDB-377)
```

Comment:
I am not sure we should add the changelog item when it is not yet visible to our users. I would probably add one item when all the support from `bzimport` through model and API to `BBSync` is done.

Reply:
Makes sense, will remove for now